src/options.cpp

```diff
@@ -1659,23 +1674,25 @@ void options_manager::show(bool ingame)
         }
 
         draw_scrollbar(w_options_border, iCurrentLine, iContentHeight,
-                       mPageItems[iCurrentPage].size(), iTooltipHeight + 2, 0, BORDER_COLOR);
+                       mPageItems[iCurrentPage].size(), iTooltipHeight + 2 + iWorldOffset, 0, BORDER_COLOR);
         wrefresh(w_options_border);
 
         //Draw Tabs
-        mvwprintz(w_options_header, 0, 7, c_white, "");
-        for (int i = 0; i < (int)vPages.size(); i++) {
-            if (!mPageItems[i].empty()) { //skip empty pages
-                wprintz(w_options_header, c_white, "[");
-                if ( ingame && i == iWorldOptPage ) {
-                    wprintz(w_options_header,
-                            (iCurrentPage == i) ? hilite(c_ltgreen) : c_ltgreen, _("Current world"));
-                } else {
-                    wprintz(w_options_header, (iCurrentPage == i) ?
-                            hilite( c_ltgreen ) : c_ltgreen, "%s", _( vPages[i].second.c_str() ) );
+        if ( !world_options_only ) {
+            mvwprintz(w_options_header, 0, 7, c_white, "");
+            for (int i = 0; i < (int)vPages.size(); i++) {
+                if (!mPageItems[i].empty()) { //skip empty pages
```

Comment:
You could invert the `if` to save on nesting.
The number of indents here is huge, squeezing the code against the right margin.